templates/CRM/Grant/Form/Task/Update.tpl

```diff
@@ -23,23 +23,18 @@
  | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
  +--------------------------------------------------------------------+
 *}
-{* Update Grants *}
+{* Update Grants Via Search Actions *}
 <div class="crm-block crm-form-block crm-grants-update-form-block">
     <p>{ts}Enter values for the fields you wish to update. Leave fields blank to preserve existing values.{/ts}</p>
     <table class="form-layout-compressed">
         {* Loop through all defined search criteria fields (defined in the buildForm() function). *}
         {foreach from=$elements item=element}
             <tr class="crm-contact-custom-search-form-row-{$element}">
                 <td class="label">{$form.$element.label}</td>
-                {if $element eq 'decision_date'}
-                    <td>{include file="CRM/common/jcalendar.tpl" elementName=decision_date}<br />
-                    <span class="description">{ts}Date on which the grant decision was finalized.{/ts}</span></td>
-                {else}
-                    <td>{$form.$element.html}</td>
-                {/if}
+                <td>{$form.$element.html}</td>
             </tr>
         {/foreach}
     </table>
     <p>{ts 1=$totalSelectedGrants}Number of selected grants: %1{/ts}</p>
     <div class="crm-submit-buttons">{include file="CRM/common/formButtons.tpl" location="bottom"}</div>
-</div><!-- /.crm-form-block -->
+</div>
```

Comment:
Just for my information, what was this comment that has been removed originally for? Is it a cut and paste remnant?

Reply:
For whatever reason, the original .tpl code was extremely messy, lines were indented completely at random and it was very hard to tell what opening tags corresponded to what closing tags. So I guess rather than indent properly someone decided to put a comment whenever closing a tag about what opening tag it corresponded to?

Reply:
Since Grants is a newer component I assume this tpl was created by copy-pasting some other tpl and thus inherited messy whitespace and useless closing comments.